third_party/2and3/werkzeug/exceptions.pyi

```diff
@@ -1,152 +1,153 @@
 from typing import Any, Dict, NoReturn
 
+from werkzeug.wrappers import Response
+
 class HTTPException(Exception):
-    code = ...  # type: Any
-    description = ...  # type: Any
-    response = ...  # type: Any
-    def __init__(self, description=None, response=None): ...
+    code = ...  # type: int
```

Comment:
Actually `Optional[int]`. It looks like it's `None` on this class and set to specific values in subclasses; `Optional[int]` expresses that.